cms/envs/common.py

```diff
@@ -915,7 +915,6 @@
     # other apps that are.  Django 1.8 wants to have imported models supported
     # by installed apps.
     'lms.djangoapps.verify_student',
-    'lms.djangoapps.grades.apps.GradesConfig',
```

Comment:
Because of this line, the reindexing command that runs with cms settings would hit [this line](https://github.com/edx/edx-platform/blame/master/cms/envs/common.py#L918), importing the grades signal handlers, which in turn import the grades tasks, which uses the new `RECALCULATE_GRADES_ROUTING_KEY` settings in a decorator, causing the reindexing command to fail because cms settings do not define that variable.

It seems to me that we'd be better off de-coupling lms and cms, rather than defining everything grades needs to work in cms settings. Open for debate on how to solve this issue properly, but this fixes it for the time being so I can test the new routing key on a sandbox.